programs/staking/src/lib.rs

```diff
@@ -261,4 +306,21 @@ pub enum VaultError {
     ZeroRewardAmount,
     #[msg("Withdraw amount cannot be zero")]
     ZeroWithdrawAmount,
+    #[msg("Math operation overflow")]
+    MathOverflow,
+    #[msg("LockedRewardDegradation is invalid")]
+    InvalidLockedRewardDegradation,
 }
+
+
+#[event]
+pub struct EventStake {
```

Comment:
I think we do not need the `Event` prefix. What do you think ?

Reply:
Because we declare event in the same module with context, so we cannot use the same name Stake for the event